test/.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
We don't want this in the test dir, only the root dir.

Reply:
Deleted